packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -116,6 +118,12 @@ class _DropdownMenuState<T> extends State<_DropdownMenu<T>> {
   CurvedAnimation _fadeOpacity;
   CurvedAnimation _resize;
 
+  // On the web, enter doesn't select things, *except* in a <select>
+  // element, which is what a dropdown emulates.
+  static final Map<LogicalKeySet, Intent> _webShortcuts =<LogicalKeySet, Intent>{
```

Comment:
Probably not critical, but this does mean that non-web versions of the app will have this taking up memory, right?

Reply:
Yes, I guess so: it's not a lot though, since it's static.  I'll make it predicated on `kIsWeb` too and it'll take up less space then.

Reply:
Actually, since it's a private static, and all uses are within `if (kIsWeb){}` blocks that will be elided, I'll bet it disappears entirely on non-web platforms.